plugins/module_utils/bonsai.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import (
+    errors, http,
+)
+
+
+def get(path):
+    url = "https://bonsai.sensu.io/api/v1/assets/{0}".format(path.lstrip("/"))
```

Comment:
Do we need the `lstrip` part? Apart from the unit tests where `path` comprises a leading `/`, `get` is always called with path argument without it. I mean, it certainly doesn't hurt if we keep it, but maybe it's not required.

Reply:
We do not. I do not even remember adding it, so let us simplify this.